### PR TITLE
ovs: add OWNERS file

### PR DESCRIPTION
**Pull request by dcbw**
We might as well own this.

@knobunc @eparis @openshift/networking

### CODE REVIEW

File: pkg/util/ovs/OWNERS [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+  - pravisankar
+  - dcbw
+  - smarterclayton
+  - rajatchopra
+  - danwinship
+  - knobunc
+approvers:
+  - pravisankar
+  - dcbw
+  - smarterclayton
```
Comment (deads2k) [inline pkg/util/ovs/OWNERS]: top level owners don't have to explicitly called out.

File: pkg/util/ovs/OWNERS [hunk 2]
```diff
@@ -0,0 +1,15 @@
+reviewers:
+  - pravisankar
+  - dcbw
+  - smarterclayton
```
Comment (deads2k) [inline pkg/util/ovs/OWNERS]: Unless he actually authored/directly owns the code in the package, you can leave him off of reviewers, to leave first level review to the core team that owns it.

**Comment by deads2k** [comment]
nits

/approve

**Comment by dcbw** [comment]
@deads2k thanks, removed Clayton since he doesn't author/own code in pkg/util/ovs.

**Pull request merged**